west.yml

```diff
@@ -188,7 +188,7 @@ manifest:
       groups:
         - hal
     - name: hal_nordic
-      revision: cb7600a1be4c8b177867e6d463729c07dd3f6d73
+      revision: c1c448df1b751c1c03229a447a2216f29e6aaf99
```

Comment:
A few more commits are getting pulled in addition to the doxygen fixes. Do we need those?

Reply:
Yeah, I don't think we can help that, but we close this PR in favour of https://github.com/zephyrproject-rtos/zephyr/pull/80583/files as #80583 has both manifest update that supersedes this and other fixes.